p4runtime-spec-next/P4Runtime-Spec.adoc

```diff
@@ -5526,7 +5529,7 @@ when reading the config. If the config is installed with a mechanism other than
 
 If a P4Runtime server supports both `SetForwardingPipelineConfig` as well as
 returning the `p4_device_config`, there should be read-write symmetry between
-`SetForwardingPipelineConfig` and `GetForwardingPipelineConfig` RPCs.
+`SetForwardingPipelineConfig` /a>and `GetForwardingPipelineConfig` RPCs.
```

Comment:
Is this change intentional?  If so, what is `/a>and` AsciiDoc markup for?

If this is a new one not mentioned in this file yet: https://github.com/jafingerhut/learn-asciidoc/blob/main/p4-spec-next/README-madoko-to-asciidoc-notes.md

it might be nice to add it.

Reply:
Hi Andy ,

I believe that this is a typo , I'm sorry about it.